cms/templates/export.html

```diff
@@ -6,18 +6,25 @@
   from django.utils.translation import ugettext as _
   import json
 %>
-<%block name="title">${_("Course Export")}</%block>
+<%block name="title">
+%if library:
```

Comment:
Agreeing with @sarina. Object names should typically be all lowercase when they occur inline in sentences. When they occur in titles that are title case they can have initial caps.